docs/public-builds.md

```diff
@@ -0,0 +1,53 @@
+# The PHP Manual builds
+
+The PHP Manual is written in [DocBook][docbook] and built by [PhD][phd], and
+these builds are rsynced to the web servers for users to use.
+
+## Mirror builds
+The [rsync box][rsync.php.net] builds the manuals every night, at around 05:00 UTC.
+The web servers then pick up these builds when they sync, which usually happens every hour.
+
+## Doc server builds
+The [docs development server][docs.php.net] builds the manual six times a day
+(every 4 hours, starting at midnight UTC). This takes place on the [euk2][euk2] server.
+An easy way to see when each translation was last built, is to look at the
+[doc downloads page with dates][download-docs]. Also note that several old
+translations reside on this particular server, as it attempts to build every
+translation (both active and inactive).
+
+## CHM builds
+The CHM version of the manual is built on a Windows machine and pulled on Fridays,
+for distribution to web servers. [Richard][rquadling] maintains these builds.
```

Comment:
This person does not maintain them anymore, I don't even know who does and if we still do them.

Reply:
I suggest to not mention anybody in particular. Instead use some general mail address or mailing list.